README.md

```diff
@@ -39,7 +33,329 @@ to account for, etc.
 
 `remote-collection` is here to help with that!
 
-### Usage
+## RemoteCollection
+
+Creates a new `RemoteCollection` of a generic `Resource` type. If you pass in
+another `RemoteCollection<Resource>` to the constructor, it will merge the
+resources.
+
+### Signature
+```ts
+export default class Collection<Resource extends { [key: string]: any }> {
+  constructor(fromCollection?: RemoteCollection<Resource>) {
```

Comment:
can you close out the brackets?
e.g. 
```
export default class Collection<Resource extends { [key: string]: any }> {
  constructor(fromCollection?: RemoteCollection<Resource>) {
    ...
  }
  ...
}
```

Reply:
⚡️ https://github.com/ca-la/remote-collection/pull/11/commits/8028c2c27a0cc1fbfcd4738b56df16f95ade2081
Changed this up a bit and added some example docs.